cli-feed-v3.json

```diff
@@ -104,24 +104,14 @@
       "sha2": "A148DC9C2783D2E7E5E50F59BE2EAEA7948213A30876F41171CF20E9F84754F2",
       "FUNCTIONS_EXTENSION_VERSION": "beta"
     },
-    "2.2.0": {

```

Comment:
Should we delete 2.2.0. It is supposed to be additive?

Reply:
I wasn't sure if I should remove it or not since it never actually went out.  But I'll revert it.